terraform/macos.tf

```diff
@@ -5,11 +5,13 @@ variable "host_id" {
 }
 
 locals {
+    macos_version = startswith(var.envoy_version, "1.26") ? 11 : 12
```

Comment:
I think we need 11 for 1.28 only

Reply:
Hmm actually it looks like we use 12 for everything up to and including 1.28

Reply:
I think build success on 1.26/1.27 with 12. I would switch in this case to 12 because it's faster to start